.github/workflows/lint-and-test.yaml

```diff
@@ -0,0 +1,39 @@
+name: Lint and tests
+
+on: pull_request
+
+jobs:
+  need-test:
+    runs-on: ubuntu-latest
+    outputs:
+      charts: ${{ steps.list-changed.outputs.changed }}
+    steps:
+      - uses: actions/checkout@v2
+        with:
+          fetch-depth: 0
+      - uses: helm/chart-testing-action@v2.1.0
+      - id: list-changed
+        run: |
+          changed=$(ct --config .github/ct.yaml list-changed)
+          if [[ -n "$changed" ]]; then
+            echo "changed=true" >> $GITHUB_OUTPUT
+          fi
+
+  lint-test:
+    runs-on: ubuntu-latest
+    needs: need-test
+    if: ${{ needs.need-test.outputs.charts == 'true' }}
+    steps:
+      - uses: actions/checkout@v2
+        with:
+          fetch-depth: 0
+      - uses: helm/chart-testing-action@v2.1.0
+      - run: |
+          helm plugin install https://github.com/quintush/helm-unittest
+      - uses: manusa/actions-setup-minikube@v2.4.3
+        with:
+          minikube version: v1.20.0
+          kubernetes version: v1.25.0
```

Comment:
This is what renovate is targeting, right?

Reply:
Yes.